sonar-web-plugin/src/main/java/org/sonar/plugins/web/checks/sonar/PasswordAutocompleteCheck.java

```diff
@@ -28,8 +28,10 @@
   key = "S1443",
   name = "\"autocomplete\" should be set to \"off\" on input elements of type \"password\"",
   priority = Priority.CRITICAL,
-tags = {RuleTags.HTML5, RuleTags.SECURITY})
+  status = RuleTags.STATUS_DEPRECATED,
+  tags = {RuleTags.HTML5, RuleTags.SECURITY})
 @SqaleConstantRemediation("5min")
+@Deprecated
 public class PasswordAutocompleteCheck extends AbstractPageCheck {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)